core/cautils/scaninfo.go

```diff
@@ -214,7 +215,13 @@ func (scanInfo *ScanInfo) SetPolicyIdentifiers(policies []string, kind apisv1.No
 		if !scanInfo.contains(policy) {
 			newPolicy := PolicyIdentifier{}
 			newPolicy.Kind = kind
-			newPolicy.Name = policy
+			// control can be identified only by it's id.
+			if kind == apisv1.KindControl {
```

Comment:
please see the comment I added to the structure

Reply:
Done